packages/terminal/src/widget.ts

```diff
@@ -247,9 +248,32 @@ export class Terminal extends Widget implements ITerminal.ITerminal {
       });
     });
 
-    this._term.on('title', (title: string) => {
+    term.on('title', (title: string) => {
       this.title.label = title;
     });
+
+    term.attachCustomKeyEventHandler(event => {
+      if (
+        (event.ctrlKey || event.metaKey) &&
```

Comment:
This blocks using Ctrl+C on a mac unilaterally. Instead, I think we should do a mac test, like w do in https://github.com/jupyterlab/jupyterlab/blob/ce4c47a99296721ca38ae2727737ae95b4b0216c/packages/filebrowser/src/listing.ts#L1241

Reply:
(note that we can use phosphor's IS_MAC instead of defining our own if we import Platform from phosphor's domutils)

Reply:
Thanks for the updates. Now we can take out `|| event.metaKey`